it/uk/gov/hmrc/partnershipidentificationfrontend/assets/TestConstants.scala

```diff
@@ -78,9 +79,13 @@ object TestConstants {
   )
 
   val testDefaultWelshJourneyConfig: JourneyConfig = testDefaultJourneyConfig.copy(
-    pageConfig = testDefaultPageConfig.copy(optLabels = Some(JourneyLabels(welshServiceName = "This is a welsh service name from Journey labels")))
+    pageConfig = testDefaultPageConfig.copy(optLabels = Some(JourneyLabels(optWelshServiceName = Some("This is a welsh service name"), None))
+    )
   )
 
+  val testWelshServiceName: String = "This is a welsh service name"
```

Comment:
This could be used in above config as well

Reply:
I am not sure why but if I do this, it breaks a few tests

Reply:
It worked for me if I moved the testWelshServiceName val above the config I was trying to use it in. Does that work for you?